src/components/VItemGroup/VItem.vue

```diff
@@ -82,7 +83,7 @@ import VButton from "~/components/VButton.vue"
 import VIcon from "~/components/VIcon/VIcon.vue"
 import { VPopoverContentContextKey } from "~/components/VPopover/VPopoverContent.vue"
 
-import checkmark from "~/assets/icons/checkmark.svg"
+import itemIndicator from "~/assets/icons/item-indicator.svg"
```

Comment:
Why is this different from the regular checkmark icon used in the checkboxes? The Figma component points to the same SVG for both.

Reply:
If I remember correctly, it was because of the different sizes of the checkmark. The new one is 24x24. I can look into it tomorrow.